src/eval_iter_loop.ipynb

```diff
@@ -0,0 +1,579 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 🍫 Building and Evaluate a RAG pipeline with Fondant\n",
+    "\n",
+    "> ⚠️ Please note that this notebook **is not** compatible with **Google Colab**. To complete the tutorial, you must \n",
+    "> initiate Docker containers. Starting Docker containers within Google Colab is not supported.\n",
+    "\n",
+    "This repository demonstrates a complete RAG pipeline. \n",
+    "\n",
+    "The RAG pipeline is composed of a Fondant pipeline that ingests text data into a vector database (indexing pipeline), and a Fondant pipeline that proceeds retrieval on the vector database and evaluates it (evaluation pipeline). The score is then displayed to observe the performance of the chosen RAG configuration.\n",
+    "\n",
+    "Additionally, we provide a Docker Compose setup for Weaviate, enabling local testing and\n",
+    "development, and an evaluation dataset containing questions over the first 1000 rows of the [loaded text data](https://huggingface.co/datasets/wikitext)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> ⚠️ Pay attention to the \"TODO\"s in the notebook as they require your input to run the pipelines."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Why Fondant for RAG?\n",
```

Comment:
@RobbeSneyders what do you think about this section? I think it is important for the user to understand why Fondant is used there since the pipeline we show is very basic. But maybe this kind of explanation must be elsewhere? (in a post, in the docs or somewhere else).